Formula/coredns.rb

```diff
@@ -0,0 +1,107 @@
+require "language/go"
+
+class Coredns < Formula
+  desc "DNS server that chains plugins"
+  homepage "https://coredns.io"
+  url "https://github.com/coredns/coredns/archive/v0.9.9.tar.gz"
+  sha256 "9ca565bfbd708bd9bc66db20ec2feb3cd2a3354cf8590ba99bf2871fb847413b"
+  head "https://github.com/coredns/coredns.git"
+
+  def default_coredns_config; <<~EOS
+    . {
+      hosts {
+        fallthrough
+      }
+      proxy . 8.8.8.8:53 8.8.4.4:53 {
+        protocol https_google
+      }
+      cache
+      errors
+    }
+    EOS
+  end
+
+  depends_on "go" => :build
+
+  go_resource "github.com/mholt/caddy" do
```

Comment:
`go_resource` blocks should not be needed because upstream has a `vendor` directory. If the `vendor` directory is incomplete, please file an upstream issue report requesting that it be completed, as we don't allow `go_resource` blocks in new formulae.

Reply:
They had reasons why they removed vendoring for these packages:
- coredns/coredns#696
- coredns/coredns#792
- coredns/coredns#793
- coredns/coredns#794

Is there another way you would prefer it done?

Reply:
@Lugoues they'll need to implement some kind of vendoring before this can be accepted into Homebrew, I'm afraid.